### PR TITLE
reformat summary commands

### PR DESCRIPTION
**Pull request by leoguzman**
---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Comment by leoguzman** [comment]
@RNHTTR @jedcunningham

**Review by jedcunningham — APPROVED** [review]

File: contributing-docs/10_working_with_git.rst [hunk 1]
```diff
@@ -191,11 +191,13 @@ Summary
...
-``git checkout my-branch``
-``git rebase HASH --onto apache/main``
-``git push --force-with-lease``
+```
+git fetch --all
+git merge-base my-branch apache/main
+git checkout my-branch
+git rebase HASH --onto apache/main
+git push --force-with-lease
+```
```
Comment (jedcunningham) [inline contributing-docs/10_working_with_git.rst]:
```suggestion
``
git fetch --all
git merge-base my-branch apache/main
git checkout my-branch
git rebase HASH --onto apache/main
git push --force-with-lease
``
```

**Review by jedcunningham — DISMISSED** [review]

**Review by RNHTTR — CHANGES_REQUESTED** [review]
RST doesn't render code blocks the same way as markdown. You can go to the "Files changed" tab then click the three dots then "View File" to see a preview of the file.

File: contributing-docs/10_working_with_git.rst (hunk 1 above)
Comment (uranusjr) [reply contributing-docs/10_working_with_git.rst]:
```suggestion
::

    git fetch --all
    git merge-base my-branch apache/main
    git checkout my-branch
    git rebase HASH --onto apache/main
    git push --force-with-lease
```

I think this is the correct way

https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks

**Review by RNHTTR — APPROVED** [review]
RST doesn't render code blocks the same way as markdown. You can go to the "Files changed" tab then click the three dots then "View File" to see a preview of the file.

**Review by amoghrajesh — APPROVED** [review]

**Pull request merged**